Source/Scene/Model.js

```diff
@@ -287,6 +327,8 @@ define([
      * @param {Object} [options.id] A user-defined object to return when the model is picked with {@link Scene#pick}.
      * @param {Boolean} [options.allowPicking=true] When <code>true</code>, each glTF mesh and primitive is pickable with {@link Scene#pick}.
      * @param {Boolean} [options.asynchronous=true] Determines if model WebGL resource creation will be spread out over several frames or block until completion once all glTF files are loaded.
+     * @param {Boolean} [options.instanced=false] Whether the model will be instanced.
```

Comment:
Since users will use `ModelInstanceCollection`, let's not document these...but, below, document that they are undocumented. ha.